site/package.json

```diff
@@ -11,31 +11,32 @@
   },
   "homepage": "https://github.com/Submitty/Submitty#readme",
   "dependencies": {
+    "@codemirror/autocomplete": "6.4.2",
+    "@codemirror/commands": "6.2.4",
+    "@codemirror/lang-cpp": "6.0.2",
+    "@codemirror/lang-java": "6.0.1",
+    "@codemirror/lang-json": "6.0.1",
+    "@codemirror/lang-python": "6.1.2",
+    "@codemirror/language": "6.6.0",
+    "@codemirror/language-data": "6.3.1",
+    "@codemirror/legacy-modes": "6.3.2",
+    "@codemirror/lint": "6.2.0",
+    "@codemirror/search": "6.4.0",
+    "@codemirror/state": "6.2.0",
+    "@codemirror/theme-one-dark": "6.1.2",
+    "@codemirror/view": "6.9.2",
     "@fortawesome/fontawesome-free": "6.4.0",
     "@highlightjs/cdn-assets": "11.8.0",
     "@submitty/pdf-annotate.js": "2.5.1",
     "bootstrap": "4.6.0",
     "chosen-js": "1.8.7",
     "codemirror": "5.65.13",
     "codemirror-spell-checker": "1.1.2",
-    "@codemirror/state": "6.2.0",
-    "@codemirror/view": "6.9.2",
-    "@codemirror/language": "6.6.0",
-    "@codemirror/autocomplete": "6.4.2",
-    "@codemirror/commands": "6.2.4",
-    "@codemirror/lint": "6.2.0",
-    "@codemirror/search": "6.4.0",
-    "@codemirror/theme-one-dark": "6.1.2",
-    "@codemirror/legacy-modes": "6.3.2",
-    "@codemirror/lang-json": "6.0.1",
-    "@codemirror/lang-python": "6.1.2",
-    "@codemirror/lang-java": "6.0.1",
-    "@codemirror/lang-cpp": "6.0.2",
-    "@codemirror/language-data" : "6.3.1",
     "flatpickr": "4.6.13",
     "jquery": "3.6.4",
     "jquery-ui-dist": "1.13.2",
     "jquery.are-you-sure": "1.9.0",
+    "js-cookie": "^3.0.5",
```

Comment:
Make sure to pin the exact version as we prefer to do that with all of our dependencies